account_tax_balance/models/account_tax.py

```diff
@@ -26,10 +26,11 @@ class AccountTax(models.Model):
 
     def get_context_values(self):
         context = self.env.context
+        actual_company_id = context.get("company_id", self.env.company.id)
```

Comment:
Not sure about the specific code, but I think you will get the idea:

```suggestion
        actual_company_ids = context.get("company_ids", self.env.companies.ids)
```

Reply:
`actual_company_id` is for retro-compatibility, in case someone is still using `company_id` in context

Reply:
Then:
```suggestion
        actual_company_id = context.get("company_id", False)
        ...
            context.get("company_ids", actual_company_id and [actual_company_id] or self.env.companies.ids),
```

Reply:
Why?

Reply:
For having the same default that you have put in the field.

Reply:
I don't understand. With this line I'm keeping the same behavior as it was before, in case `company_ids` and `company_id` are not passed in the context.

Reply:
OK, maybe I'm not understanding the use of this, so let's move on!